requirements.txt

```diff
@@ -0,0 +1,26 @@
+dateparser
+django~=3.2.14
+django-hstore
+django-markdownx @ git+https://github.com/alphatownsman/django-markdownx.git@e69480c64ad9c5d0499f4a8625da78cf2bb7691b
+django-sass
+django-rq
+django-simple-history
+django-hijack
+django-user-messages
+django-slack
+meilisearch
+easy-thumbnails
+lxml
+openpyxl
+psycopg2
+requests
+filetype
+setproctitle
+tqdm
+opencc
+dnspython
+typesense
+markdownify
+sentry-sdk
+gitpython
```

Comment:
Same seemingly unused package

Reply:
typesense and meilisearch are both supported.

Reply:
and I'm using typesense in prod right now for performance reason.

Reply:
markdownify is required to import review (html -> md)

Reply:
sentry-sdk and gitpython is enabled only in my prod settings to send stack track to sentry.io when 500.

Reply:
I don't why I only wanted to comment on one line but it all turned out to be multiple lines... anyway thank you for the explaination